openwrt-openwisp-monitoring/files/monitoring.init

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh /etc/rc.common
+
+START=99
+STOP=15
+USE_PROCD=1
+PROG="/usr/sbin/openwisp_monitoring"
+PROG_NAME="OpenWISP monitoring daemon"
+
+start_service() {
+		config_load openwisp
+		base_url=$(config_get http url)
+		uuid=$(config_get http uuid)
+		key=$(config_get http key)
+		verify_ssl=$(config_get http verify_ssl)
+		respawn_threshold=$(config_get http respawn_threshold)
+		respawn_timeout=$(config_get http respawn_timeout)
+		respawn_retry=$(config_get http respawn_retry)
+		config_load monitoring
+		interval=$(config_get monitoring interval)
+		monitored_interfaces=$(config_get monitoring monitored_interfaces)
+		if [ $base_url ]; then base_url="--url $base_url"; fi
+		if [ $uuid ]; then uuid="--uuid $uuid"; fi
+		if [ $key]; then key="--key $key"; fi
+		if [ $verify_ssl ]; then verify_ssl="--verify_ssl $verify_ssl"; fi
+		if [ $interval ]; then interval="--verify_ssl $verify_ssl"; fi
+		if [ $monitored_interfaces ]; then monitored_interfaces="--monitored_interfaces $monitored_interfaces"; fi
+        procd_open_instance
```

Comment:
Try passing the [instance name argument ](https://openwrt.org/docs/guide-developer/procd-init-scripts#defining_service_instances)to `procd_open_instance`, it will make the logs look nicer and helps keep things organised if we ever in the future want multiple instances to run simultaneously. `"openwisp_monitoring_${1}"`. See[ usage here](https://github.com/openwrt/packages/blob/791892d482c2255c8681b51bb0b78aa06d48f0e0/utils/watchcat/files/watchcat.init#L94).

Do we want to also restart the service when the config file is changed?